vast/integration/reference/use-schema-time-unit-when-converting-from-a-double-to-a-duration/step_00.ref

```diff
@@ -0,0 +1,2 @@
+{"StartTime": null, "Flgs": null, "Proto": null, "SrcAddr": null, "Sport": null, "Dir": null, "DstAddr": null, "Dport": null, "TotPkts": null, "TotBytes": null, "State": null, "SrcId": null, "Rank": null, "LastTime": null, "Trans": null, "Seq": null, "Dur": null, "RunTime": null, "IdleTime": null, "Mean": null, "StdDev": null, "Sum": null, "Min": null, "Max": null, "SrcMac": null, "DstMac": null, "SrcOui": null, "DstOui": null, "sTos": null, "dTos": null, "sDSb": null, "dDSb": null, "sCo": null, "dCo": null, "sTtl": null, "dTtl": null, "sHops": null, "dHops": null, "sIpId": null, "dIpId": null, "sMpls": null, "dMpls": null, "AutoId": null, "sAS": null, "dAS": null, "iAS": null, "Cause": null, "NStrok": null, "sNStrok": null, "dNStrok": null, "SrcPkts": null, "DstPkts": null, "SrcBytes": null, "DstBytes": null, "TotAppByte": null, "SAppBytes": null, "DAppBytes": null, "PCRatio": null, "Load": null, "SrcLoad": null, "DstLoad": null, "Loss": null, "SrcLoss": null, "DstLoss": null, "pLoss": null, "pSrcLoss": null, "pDstLoss": null, "Retrans": null, "SrcRetra": null, "DstRetra": null, "pRetran": null, "pSrcRetr": null, "pDstRetr": null, "SrcGap": null, "DstGap": null, "Rate": null, "SrcRate": null, "DstRate": null, "SIntPkt": "20.0ns", "SIntDist": null, "SIntPktAct": null, "SIntActDist": null, "SIntPktIdl": null, "SIntIdlDist": null, "SIntPktMax": null, "SIntPktMin": null, "DIntPkt": null, "DIntDist": null, "DIntPktAct": null, "DIntActDist": null, "DIntPktIdl": null, "DIntIdlDist": null, "DIntPktMax": null, "DIntPktMin": null, "SrcJitter": null, "SrcJitAct": null, "SrcJitIdl": null, "DstJitter": null, "DstJitAct": null, "DstJitIdl": null, "Label": null, "srcUdata": null, "dstUdata": null, "SrcWin": null, "DstWin": null, "sVlan": null, "dVlan": null, "sVid": null, "dVid": null, "sVpri": null, "dVpri": null, "SRange": null, "ERange": null, "SrcTCPBase": null, "DstTCPBase": null, "TcpRtt": null, "SynAck": null, "AckDat": null, "TcpOpt": null, "Inode": null, "Offset": null, "sMeanPktSz": null, "dMeanPktSz": null, "sPktSz": null, "sMaxPktSz": null, "dPktSz": null, "dMaxPktSz": null, "sminPktSz": null, "dminPktSz": null, "STcpMax": null, "DTcpMax": null, "dlDur": null, "dlsTime": null, "dllTime": null, "dsPkts": null, "ddPkts": null, "dsBytes": null, "ddBytes": null, "JDelay": null, "LDelay": null, "sEnc": null, "dEnc": null, "IcmpId": null, "schema": "record"}
```

Comment:
Please change the test for this to only show the relevant columns by putting a `select SIntPkt` into the pipeline. It's otherwise really hard to parse this output.

Reply:
good idea